fhir-config/src/main/java/com/ibm/fhir/config/ResourcesConfigAdapter.java

```diff
@@ -0,0 +1,114 @@
+/*
+ * (C) Copyright IBM Corp. 2022
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package com.ibm.fhir.config;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.LinkedHashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import java.util.stream.Collectors;
+
+import com.ibm.fhir.config.PropertyGroup.PropertyEntry;
+import com.ibm.fhir.core.ResourceTypeName;
+
+/**
+ * An abstraction for the ibm-fhir-server fhirServer/resources property group
```

Comment:
indicate that we should create a new instance of this for each request (at least).  otherwise the property lookups won't be dynamic when retrieved through here